docs/02-app/01-building-your-application/04-styling/03-css-in-js.mdx

```diff
@@ -14,6 +14,7 @@ description: Use CSS-in-JS libraries with Next.js
 The following libraries are supported in Client Components in the `app` directory (alphabetical):
 
 - [`kuma-ui`](https://kuma-ui.com)
+- [Material UI](https://mui.com/material-ui/guides/next-js-app-router/)
```

Comment:
```suggestion
- [`@mui/material`](https://mui.com/material-ui/guides/next-js-app-router/)
```

Should we use the package name here to be consistent with the others in this list?